plugins/idamagic/helpers.py

```diff
@@ -162,10 +175,10 @@ def to_bool(param, default=False):
         # This will happen when param isn't a string
         pass
 
-    if param in {1, "1", "true", "yes", "y", True}:
```

Comment:
Just curious why the int was removed here and below

Reply:
This is another case of VSCode complaining. It isn't a necessary change, but you can see why having both `1` and `True` in the set would not be needed:

```
>>> a = 1
>>> set = {True}
>>> a in set
True
```